tests/mypy/modules/success.py

```diff
@@ -127,3 +128,10 @@ def foo(a: int, *, c: str = 'x') -> str:
 
 foo(1, c='thing')
 foo(1)
+
+if sys.version_info < (3, 7):
```

Comment:
I think you can just remove this check and test the `ForwardRef` for all python versions. Otherwise the test looks good

Reply:
Yea, I thought about it, but I think we have no need in testing build-ins so I made test only for implemented functionality

Reply:
I completely agree in general but I feel like this is one of the cases where it's worth testing it as the implemented code is here to mimic the builtin. But I feel like it's more a matter of taste than anything else ;)

Reply:
Yup, matter of taste, let's wait for anyone's else opinion then ;)

Reply:
@PrettyWood ForwardRef `__init__` signature different between versions: 
```python
def __init__(self, arg, is_argument=True):
```
for python 3.8, so we need make more complex test if we want cover all cases :)

Reply:
I think best to remove this version check.

from what I can see the test here should still work with 3.8?